fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2030,6 +2030,13 @@ public class Config extends ConfigBase {
      */
     @ConfField
     public static int cloud_native_meta_port = 6090;
+
+    /**
+     *  Whether volume can be created from conf. If it is enabled, a builtin storage volume may be created.
+     */
+    @ConfField
+    public static boolean enable_volume_from_conf = true;
+
     // remote storage related configuration
     @ConfField(comment = "storage type for cloud native table. Available options: \"S3\", \"HDFS\", \"AZBLOB\". case-insensitive")
     public static String cloud_native_storage_type = "S3";
```

Comment:
shall set this to empty by default.

Reply:
some users may use the default value s3 in 3.0 version, so we should not change the default value.